### PR TITLE
3 - changes in indentation pattern and separation of the functions in…

### PR DESCRIPTION
**Pull request by robertoramosjr**

### CODE REVIEW

File: project_script/constantvalues.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+electron_charge = 1.6*10**(-19)
+
+device_area = 1.4*10**(-5)
+
+device_thickness = 600*10**(-9)
+
+current_correction_factor = 1*10**(-9)
+
+charge_density_correction_factor = 1*10**(-6)
```
Comment (caioalexandre) [inline project_script/constantvalues.py:9]: In other languages, normally the constants is written in CAPS, to easily show that is a constant. Check if this practice is comon in python too.

File: project_script/main.py [hunk 2]
```diff
@@ -1,108 +1,105 @@
...
-
-
-def separate_odd_columns(data_frame):
-    return data_frame.iloc[:, [i for i in range(len(data_frame.columns)) if i % 2 == 0]]
-
-
-def is_valid_file(file_path):
-    return True if file_path != "" and os.path.exists(file_path) else False
+import datatreatment as dt
+import constantvalues as cv
```
Comment (caioalexandre) [inline project_script/main.py:6]: I can't remember, if it's a good practice the names all in lowercase, check pls.

File: project_script/main.py [hunk 3]
```diff
@@ -1,108 +1,105 @@
...
-
-
 def sanitize_data_frames():
     global even_columns_subtracted, time_photo_celiv_ramp_time
-    if is_bigger_data_frame(time_photo_celiv_ramp_time, even_columns_subtracted):
-        time_photo_celiv_ramp_time = equalize_data_frame_rows(time_photo_celiv_ramp_time, even_columns_subtracted)
+    if dt.is_bigger_data_frame(
+            time_photo_celiv_ramp_time,
+            even_columns_subtracted
+    ):
```
Comment (caioalexandre) [inline project_script/main.py:18]: If this line doesn't cross the limit line pattern, then doesn't need to break. But if it's do, ok, NP.

File: project_script/main.py [hunk 4]
```diff
@@ -1,108 +1,105 @@
...
+        time_photo_celiv_ramp_time = dt.equalize_data_frame_rows(
+            time_photo_celiv_ramp_time,
+            even_columns_subtracted
+        )
         return
-    even_columns_subtracted = equalize_data_frame_rows(even_columns_subtracted, time_photo_celiv_ramp_time)
+    even_columns_subtracted = dt.equalize_data_frame_rows(
+        even_columns_subtracted,
+        time_photo_celiv_ramp_time
+    )
```
Comment (caioalexandre) [inline project_script/main.py:27]: The same above.

File: project_script/main.py [hunk 5]
```diff
@@ -1,108 +1,105 @@
...
     for key, value in enumerate(even_columns_subtracted_transposed_in_arrays):
-        integrated_values.append(integrate.simps(even_columns_subtracted_transposed_in_arrays[key],
-                                                 time_photo_celiv_ramp_time_transposed[key], even='avg'))
+        integrated_values.append(
+            integrate.simps(
+                even_columns_subtracted_transposed_in_arrays[key],
+                time_photo_celiv_ramp_time_transposed[key],
+                even='avg'
+            )
+        )
```
Comment (caioalexandre) [inline project_script/main.py:40]: In this case, it's ok to break them all, it's a more complex readable algorithm.

File: project_script/main.py [hunk 6]
```diff
@@ -1,108 +1,105 @@
...
     return integrated_values
 
 
-# device_thickness = ask.ask_device_thickness()
+# device_thickness = ask.device_thickness()
+# device_area = ask.device_area()
+charge_density_calculation_constant = cv.current_correction_factor * cv.charge_density_correction_factor \
```
Comment (caioalexandre) [inline project_script/main.py:46]: The name `_constant` is it to flag that variable it's a constant, or it's something about physics? If it's to flag, i made a comment about constants above, but if the answer is no, just ignore.

File: project_script/main.py [hunk 7]
```diff
@@ -1,108 +1,105 @@
...
-                                                                   sep='\t', header=None))\
+current_dark_celiv_ramp_time = dt.separate_even_columns(
+    pd.read_table(
+        "C:/Users/robee/Desktop/2-dark-celiv-current-celiv-only-alterado.txt",
+        sep='\t',
+        header=None
+    )
+)\
     .abs()\
     .dropna()
```
Comment (caioalexandre) [inline project_script/main.py:63]: Maybe the correct indentation would be like this, right?

image

File: project_script/main.py [hunk 8]
```diff
@@ -1,108 +1,105 @@
...
+#     path_photo = ask.file_path("photo-celiv")
+
+current_photo_celiv_ramp_time = dt.separate_even_columns(
+    pd.read_table(
+        "C:/Users/robee/Desktop/3-photo-celiv-basic-current-celiv-only-alterado.txt",
+        sep='\t',
+        header=None
+    )
+)\
     .abs()
```
Comment (caioalexandre) [inline project_script/main.py:78]: The same above.

File: project_script/main.py [hunk 9]
```diff
@@ -1,108 +1,105 @@
...
 
 sanitize_data_frames()
 
-data_ramp_time = pd.concat([time_photo_celiv_ramp_time, even_columns_subtracted],
-                           axis=1).sort_index(axis=1)
+data_ramp_time = pd.concat(
+    [time_photo_celiv_ramp_time, even_columns_subtracted],
+    axis=1
+)\
+    .sort_index(axis=1)
```
Comment (caioalexandre) [inline project_script/main.py:98]: The same above. The lines 95, 96 and 97, should be one tab forward.

**Comment by caioalexandre** [comment]
In case of doubts, call me.

Congrats. You're doing a great job!

File: project_script/constantvalues.py (hunk 1 above)
Comment (robertoramosjr) [reply project_script/constantvalues.py:9]: I checked in PEP8, and it's really in CAPS. Thanks!

File: project_script/main.py (hunk 2 above)
Comment (robertoramosjr) [reply project_script/main.py:6]: According to PIP8, to python, it is!

File: project_script/main.py (hunk 3 above)
Comment (robertoramosjr) [reply project_script/main.py]: Done!

File: project_script/main.py (hunk 4 above)
Comment (robertoramosjr) [reply project_script/main.py]: Also done!

**Pull request merged**